src/components/NewMessage/NewMessagePollEditor.vue

```diff
@@ -102,39 +121,47 @@ export default {
 	emits: ['close'],
 
 	setup() {
-		return {
-			pollsStore: usePollsStore(),
-		}
-	},
-
-	data() {
-		return {
+		const pollForm = reactive({
+			question: '',
+			options: ['', ''],
 			isPrivate: false,
 			isMultipleAnswer: false,
-			pollQuestion: '',
-			pollOptions: ['', ''],
-		}
-	},
+		})
+		const isFilled = computed(() => !!pollForm.question || pollForm.options.some(option => option))
 
-	computed: {
-		isFilled() {
-			return !!this.pollQuestion || this.pollOptions.some(option => option)
-		},
+		const exportPollBlob = computed(() => {
+			if (!isFilled.value) {
+				return null
+			}
```

Comment:
```suggestion
```

Being checked in DOM

Reply:
and should be lazy computed, until isFilled is true. Removing